src/OrchardCore.Modules/OrchardCore.Lists/Controllers/OrderController.cs

```diff
@@ -55,9 +59,18 @@ public async Task<IActionResult> UpdateContentItemOrders(string containerId, int
                 return NotFound();
             }
 
+            foreach (var pagedContentItem in pageOfContentItems)
+            {
+                if (!await _authorizationService.AuthorizeAsync(User, Permissions.PublishContent, pagedContentItem))
+                {
+                    return Forbid();
+                }
```

Comment:
used forbid here based on what we are doing in https://github.com/OrchardCMS/OrchardCore/pull/5396